nixos/modules/services/video/wivrn.nix

```
@@ -84,6 +114,19 @@ in
         };
       };
 
+      extraServerFlags = mkOption {
+        type = types.listOf types.str;
+        description = "Flags to add to the wivrn service.";
+        default = [ ];
+        example = literalExpression ''[ "--no-publish-service" ]'';
```

literalExpression is only required if you reference variables like `pkgs` or `config`.
```suggestion
        example = [ "--no-publish-service" ];
```